# ML-02 — Research Question and Provisional Lane

This skeleton is yours to fill. Work the sections **in order** — each one has a one-line hint. Simple words, honest numbers.

> Working with an AI assistant? Tell it to read `skills/README.md` first and load the one skill this assignment names on its card.

## 1. My lane (or freestyle) and why

*Name your lane — or say 'freestyle' and describe your own question. One short paragraph: why this one?*

## 1. My Lane (or Freestyle) and Why

**Lane:** Refresh / Content Opportunity Scoring

I chose the Refresh / Content Opportunity Scoring lane because websites often contain thousands of pages, and reviewing every page is not practical. This project focuses on identifying which pages should be reviewed first using content and performance signals. The goal is to help content teams prioritize their work and make better decisions based on data.

## 2. The question: decision, action, cost of a wrong call

*What decision does your work improve? Who acts on it? What does a wrong recommendation cost?*

## 2. The Question: Decision, Action, Cost of a Wrong Call

**Decision:**  
The project helps decide which content pages should be reviewed first.

**Action:**  
The content or SEO team will review and update the highest-priority pages before spending time on lower-priority pages.

**Cost of a Wrong Call:**  
If a page is incorrectly marked as high priority, the team may waste time reviewing it. If an important page is missed, the website may lose an opportunity to improve its performance.

## 3. Quick look at the data (2-3 real numbers)

*Load the starter CSV below and show 2-3 real numbers that make your lane look worth the next 7 weeks.*

In [10]:
print("Total Pages:", len(df))
print("Average Impressions (90d):", round(df["impressions_90d"].mean(), 2))
print("Average Sessions (90d):", round(df["sessions_90d"].mean(), 2))
print("Average CTR:", round(df["ctr"].mean(), 4))

Total Pages: 30000
Average Impressions (90d): 5200.37
Average Sessions (90d): 37.07
Average CTR: 0.5107


## 3. Quick Look at the Data

I explored the starter dataset to understand whether it is suitable for this project.

- Total Pages: **30,000**
- Average Impressions (90 days): **5,200.37**
- Average Sessions (90 days): **37.07**
- Average CTR: **0.5107**

These statistics show that the dataset contains a large number of pages along with useful search and content performance metrics. This makes it suitable for analyzing and prioritizing which pages should be reviewed first.

## 4. Careful words: what I can and can't claim

*Write what your work will be able to say (observed, directional, decision-support) — and what it never will (causal proof, 'predicting Google').*

## 4. Careful Words: What I Can and Can't Claim

This project can identify patterns in the available data and provide decision-support recommendations for prioritizing content review. The results are based on observed signals and should be interpreted carefully.

This project cannot prove that updating a page will improve rankings or traffic. It also cannot predict Google's algorithms or establish cause-and-effect relationships.

## 4. Careful Words: What I Can and Can't Claim

This project uses the available data to identify patterns and provide decision-support recommendations for content review. The recommendations are based on observed signals and should help prioritize which pages to review first.

This project cannot prove that updating a page will improve its ranking or traffic. It also cannot predict Google's algorithms or guarantee future results. Human review is still required before making final decisions.

## Self-check

Before you submit, confirm each line honestly:

- [ ] Every section above is filled — markdown thinking AND the code that backs it
- [ ] The notebook runs top to bottom with no errors (Runtime → Run all)
- [ ] No client names, URLs, or private queries anywhere
- [ ] My claims use careful words: observed, measured, directional, decision-support
- [ ] Committed to my repo under `work/notebooks/` — then submit your repo URL on the card. Done.